<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunSetupWidget</class>
 <widget class="QWidget" name="RunSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ql_totalLength">
       <property name="text">
        <string>Total capillary length (cm)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FloatingValueLineEdit" name="qle_totalLength"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ql_detectorPosition">
       <property name="text">
        <string>Detector position (cm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="FloatingValueLineEdit" name="qle_detectorPosition"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ql_polarity">
       <property name="text">
        <string>Polarity (at inlet)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="qcbox_polarity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ql_voltage">
       <property name="text">
        <string>Driving voltage (V)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="FloatingValueLineEdit" name="qle_drivingVoltage"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>EOF</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="qcbox_eof">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ql_eofValue">
       <property name="text">
        <string>&lt;EOF VALUE&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="FloatingValueLineEdit" name="qle_eofValue"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="qcb_correctForIS">
     <property name="text">
      <string>Ionic strength correction</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/ui/elementaries//floatingvaluelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
